src/Prometheus.Client/LabelsHelper.cs

```diff
@@ -80,7 +80,7 @@ public static int GetHashCode(IReadOnlyList<string> values)
             for (var i = 0; i < values.Count; i++)
             {
                 var val = values[i];
-                if(string.IsNullOrEmpty(val))
+                if(val == null)
```

Comment:
There is a similar check in GetTupleHashCode method, need to be updated as well.